src/run_post_deploy_scripts.sh

```diff
@@ -1,31 +1,22 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
-php=$(kubectl get pods --namespace "${HELM_NAMESPACE}" \
-  --sort-by=.metadata.creationTimestamp \
-  --field-selector=status.phase=Running \
-  -l "app=wordpress-php,release=${HELM_RELEASE}" \
-  -o jsonpath="{.items[-1:].metadata.name}")
+workspace=/tmp/workspace/src
 
-for file in $(kubectl -n "${HELM_NAMESPACE}" exec "$php" -- ls ./post_deploy_scripts); do
-    echo ""
-    echo "Running the local script : $(basename "$file")"
-    echo ""
-    kubectl -n "${HELM_NAMESPACE}" exec "$php" -- bash "post_deploy_scripts/$file"
-done
+[ -d "$workspace/tasks/post-deploy" ] || {
+  echo "No post-deploy task folder found in $workspace"
+  ls -al "$workspace"
+  exit 0
+}
 
-echo "Now check the common post deploy scripts and run them as well"
+find "$workspace" -type f
 
-mkdir source
-pushd source
-git clone https://github.com/greenpeace/planet4-base-fork .
-popd
+cp -f "$workspace/post_deploy_scripts/*" "$workspace/tasks/post-deploy"
```

Comment:
This is unwanted, and maybe I wasn't clear, in the original issue description.

Instead of copying scripts from pos_deploy_scripts, we want a single source of truth for base tasks, and nro tasks, merge those two by a simple copy operation (in keeping with the approach we're using with composer), and then execute them.

I'd like to find a way to delte the post_deploy_scripts folder wherever it is used, and incorporate those scripts into `tasks/post-deploy`

Reply:
Sorry, I don't get your point here. Can you elaborate more on this?

Reply:
Delete post_deploy_scripts
Put them in tasks/post-deploy.
Consider also putting the rest of the post-deploy jobs in the Makefile into the base-fork/tasks/post-deploy

Reply:
> Consider also putting the rest of the post-deploy jobs in the Makefile into the base-fork/tasks/post-deploy

I don't see anything related here, that could be added.  Which jobs are you referring?
https://github.com/greenpeace/planet4-builder/blob/master/src/Makefile#L258

Reply:
This: https://github.com/greenpeace/planet4-builder/blob/master/src/Makefile#L243-L257

If it happens post-deploy, consider adding it to the base-fork post-deploy tasks.

Reply:
Scope creep ;)